src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseDocumentsTest.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.hmppsallocations.integration.unallocatedcases
+
+import com.fasterxml.jackson.core.type.TypeReference
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsallocations.client.Document
+import uk.gov.justice.digital.hmpps.hmppsallocations.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsallocations.integration.responses.documentsResponse
+
+class GetCaseDocumentsTest : IntegrationTestBase() {
+
+  @Test
+  fun `must return all documents associated with CRN`() {
+    val crn = "X123456"
+    documentsResponse(crn)
+    val expectedResponse = objectMapper.readValue(documentsResponse(), object : TypeReference<List<Document>>() {})
+    webTestClient.get()
+      .uri("/cases/unallocated/$crn/documents")
+      .headers { it.authToken(roles = listOf("ROLE_MANAGE_A_WORKFORCE_ALLOCATE")) }
+      .exchange()
+      .expectStatus()
+      .isOk
+      .expectBody()
+      .json(objectMapper.writeValueAsString(expectedResponse))
+  }
+
+  @Test
+  fun `cannot get all documents when no auth token supplied`() {
```

Comment:
We could do with a negative test - what happens if the CRN doesn't exist or - more likely - the int endpoint returns an error? I suspect it will throw a 500 but is that what we want?